### DISCUSSION TITLE
Is there a way to do a {{@now}} + 2 days ?

### DISCUSSION THREAD

**Discussion by lewandowski-anthony** [Decathlon/tzatziki#34] [Q&A] [answered]
Hello everyone :)
In order to tests a date, we want to generate a date which will be the current date with 2 more days added. Is it actually possible ?
Thanks a lot

**Comment by mamdouni** [comment, marked as answer]
Yes, here is an example : 
```
  Scenario: we can set the time in our tests
    Given that the current time is {{{[@2 days from now at midnight]}}}
    Then now is equal to "2022-02-18T00:00:00Z"
```
![image](https://user-images.githubusercontent.com/61866853/154248175-08adf7c6-effa-4585-abd3-9977f50c7c5e.png)
